concourse/tasks/bosh_cancel_all_tasks/run.rb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+require 'json'
```

Comment:
you may require to use the same trick to convert dns name to IP, like we do in `scripts/bosh_cli_v2_login.sh`. Because to ease user configuration, user set the dns name but bosh certificates are bound to IP.